packages/@aws-cdk/aws-cloudfront/package.json

```diff
@@ -94,9 +94,22 @@
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
   "stability": "experimental",
-  "maturity": "experimental",
+  "features": [
+    {
+      "name": "Higher level constructs for CloudFrontWebDistribution",
+      "stability": "Stable"
+    },
+    {
+      "name": "Higher level constructs for Distribution",
+      "stability": "Experimental"
+    }
+  ],
   "awslint": {
     "exclude": [
+      "props-physical-name:@aws-cdk/aws-cloudfront.Distribution",
+      "props-physical-name:@aws-cdk/aws-cloudfront.DistributionProps",
+      "props-physical-name:@aws-cdk/aws-cloudfront.CloudFrontWebDistribution",
+      "props-physical-name:@aws-cdk/aws-cloudfront.CloudFrontWebDistributionProps",
```

Comment:
why?

Reply:
There is no `DistributionName` property of a distribution. It's one of the few resources where no name parameter exists.